entities/bizaiChats.json

```diff
@@ -0,0 +1,56 @@
+{
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "uid of the chat",
+      "example": "d290f1ee-6c54-4b01-90e6-d701748f0851"
+    },
+    "actor_uid": {
+      "type": "string",
+      "description": "uid of the actor that created the chat",
+      "example": "ghp0f1ee-6c54-4b01-90e6-d701748f0851"
+    },
+    "agent": {
+      "type": "string",
+      "description": "Agent associated with the chat",
+      "example": "vanilla"
+    },
+    "metadata": {
+      "type": "object",
+      "description": "metadata of the chat",
+      "example": {
+        "directory_uid": "k98axtpqg7h1whh6",
+        "instruction": "You are a helpful assistant. Answer all questions to the best of your ability."
+      }
+    },
+    "created_at": {
+      "type": "string",
+      "description": "Date the chat was created",
+      "format": "date-time",
+      "example": "2024-05-20T14:00:00.000Z"
```

Comment:
You don't need to have an example here and also at the end of the file. you can keep only the full example at the end

Reply:
You need to have one example object at the end. This is just redundant. I will not fail the PR for this, but it just creates clutter